docs/index.html

```diff
@@ -9247,24 +9254,28 @@
 var _user$project$Bubbling_Main$CounterMsg = function (a) {
 	return {ctor: 'CounterMsg', _0: a};
 };
-var _user$project$Bubbling_Main$counter = _user$project$Glue$poly(
-	{
-		get: function (_) {
-			return _.counter;
-		},
-		set: F2(
-			function (subModel, model) {
-				return _elm_lang$core$Native_Utils.update(
-					model,
-					{counter: subModel});
-			}),
-		init: _user$project$Bubbling_Counter$init(_user$project$Bubbling_Main$Even),
-		update: _user$project$Bubbling_Counter$update(_user$project$Bubbling_Main$Even),
-		view: _user$project$Bubbling_Counter$view(_user$project$Bubbling_Main$CounterMsg),
-		subscriptions: function (_p0) {
-			return _elm_lang$core$Platform_Sub$none;
-		}
-	});
+var _user$project$Bubbling_Main$counter = A2(
+	_user$project$Glue$withView,
+	function (model) {
+		return A2(_user$project$Bubbling_Counter$view, _user$project$Bubbling_Main$CounterMsg, model.counter);
+	},
+	_user$project$Glue$poly(
+		{
+			get: function (_) {
+				return _.counter;
+			},
+			set: F2(
+				function (subModel, model) {
+					return _elm_lang$core$Native_Utils.update(
+						model,
+						{counter: subModel});
+				}),
+			init: _user$project$Bubbling_Counter$init(_user$project$Bubbling_Main$Even),
+			update: _user$project$Bubbling_Counter$update(_user$project$Bubbling_Main$Even),
+			subscriptions: function (_p0) {
+				return _elm_lang$core$Platform_Sub$none;
+			}
+		}));
 var _user$project$Bubbling_Main$subscriptions = A2(
 	_user$project$Glue$subscriptions,
 	_user$project$Bubbling_Main$counter,
```

Comment:
I was testing examples and this was builded during it, I did not pay attention, it wasn't important for my question :) But good to know 👍